ColorPicker/ColorPicker.js

```diff
@@ -208,26 +208,26 @@ const ColorPickerBase = kind({
 						>
 							{children}
 						</Group>
-						<Button icon="ellipsis" size="small" onTap={onToggleExtended} className={css.swatch} />
+						<Button aria-label={$L('More')} icon="ellipsis" size="small" onTap={onToggleExtended} className={css.swatch} />
 						<div className={css.sliders}>
 							<Row align="center">
-								<Cell>
+								<Cell aria-label={$L('Hue')} role="region">
```

Comment:
Everything looks good to me except `these aria-label`s.
It works fine. But we might use `accessibilityHint` or something like this.
Let's consider it when supporting custom aria-labels.